cnf-tests/testsuites/e2esuite/test_suite_test.go

```diff
@@ -74,6 +76,15 @@ func init() {
 	if os.Getenv("SKIP_TEST_NAMESPACES_CREATION") == "true" {
 		skipTestNSCreation = true
 	}
+
+	featuresVar := os.Getenv("FEATURES")
```

Comment:
I'd rather accept a comma separated value (and error if the given value is not a valid one). This won't catch typos.

Reply:
Also, calling the variable "FEATURES" is misleading. What that really drives is what setup / teardown, plus we may have inconsistencies between focus and FEATURES.

Ideally, focus / skip is what should drive this too. One dumb way to do so could be parse os.Args and find the gingko.focus / ginkgo.skip parameter and update the map accordingly, not sure if reading directly the argument will work too.

Reply:
but if gingko.focus is set to test name? how do you get the feature name from it? FEATURES var is always features, not tests.

Reply:
But this assumes you are calling --focus with a list of features. Otherwise,the tests would be failing because the ns is not there.
So, --focus / --skip and FEATURES (or whatever the right name is) must always be in sync, otherwise it's misconfiguration. What I'd like to see, is avoiding doing the configuration twice exposing the risk of misconfigs

Reply:
if you set --focus to a test and FEATURES to the feature name you get the ns created. but if you don't have FEATURES var but only use --focus, which is set to a test name, how do you know which ns to create?

Reply:
i introduced a new var to focus on tests (FOCUS_TESTS), so FEATURES can be just features names.

Reply:
Fair enough, thanks!